app/src/androidTest/java/com/github/freeman/bootcamp/MainMenuTest.kt

```diff
@@ -19,14 +19,20 @@ import com.github.freeman.bootcamp.ui.theme.BootcampComposeTheme
 import com.github.freeman.bootcamp.videocall.VideoCallActivity
 import com.github.freeman.bootcamp.games.wordle.WordleMenu
 import com.github.freeman.bootcamp.recorder.AudioRecordingActivity
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
+@HiltAndroidTest
 @RunWith(AndroidJUnit4::class)
 class MainMenuTest {
-    @get:Rule
+    @get:Rule(order= 0)
```

Comment:
I find it interesting that you use give a specific order to the execution of your rules as this isn't something I believe that has been used up until now.